packages/@vue/cli-plugin-typescript/lib/tslint.js

```diff
@@ -1,4 +1,4 @@
-module.exports = function lint (args = {}, api, silent) {
+module.exports = function lint(args = {}, api, silent) {
```

Comment:
Please don't remove the space before the parenthesis. It's in our style guide https://github.com/vuejs/eslint-plugin-vue-libs/blob/a3b63ec188d504e1421c9437833dd3f92fdb358a/config.js#L122

Reply:
fixed with d5a3c96